examples/flows/end-to-end-test-yaml/input_schema.yaml

```diff
@@ -0,0 +1,40 @@
+additionalProperties: false
+required:
+  - source_endpoint_id
+  - source_path
+  - destination_endpoint_id
+  - destination_path
```

Comment:
With the `globus-collection` format just about to land, it will be worthwhile to circle back around to address this. @seren do you want me to patch this to use the new format?

Reply:
That would be great, thanks.!

Reply:
I've modified this to use the `globus-collection` format. Please review!